doc/modules/linear_model.rst

```diff
@@ -888,6 +888,156 @@ to warm-starting (see :term:`Glossary <warm_start>`).
     .. [9] `"Performance Evaluation of Lbfgs vs other solvers"
             <http://www.fuzihao.org/blog/2016/01/16/Comparison-of-Gradient-Descent-Stochastic-Gradient-Descent-and-L-BFGS/>`_
 
+.. _Generalized_linear_regression:
```

Comment:
Is it time to split up this file?

Also: LogisticRegression above is a generalized linear model.

Reply:
> Is it time to split up this file?

Maybe it's time to re-organize this section, but I'm trying to keep changes in this PR to a bare minimum.

> Also: LogisticRegression above is a generalized linear model.

Yeah, but as previously mentioned, before this PR we never define what a GLM is...